pom.xml

```diff
@@ -45,11 +45,11 @@ Licensed under the MIT License.
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <javadoc.options/>
     <!-- Main dependency version numbers -->
-    <version.cassandra-driver>4.9.0</version.cassandra-driver>
+    <version.cassandra-driver>[4.7,4.11)</version.cassandra-driver>
```

Comment:
Are there any issues with 4.6 and lower?

Reply:
None known, but this is what we've tested.